### PR TITLE
add missing machineTypes for Google Cloud

### PR DESCRIPTION
**Pull request by infowolfe**
Title says it all.

### CODE REVIEW

File: src/scrapers/gcpMachineTypes.ts [hunk 1]
```diff
@@ -7,13 +7,26 @@ import { generatePriceHash, generateProductHash } from '../db/helpers';
 import { upsertProducts } from '../db/upsert';
 import { findProducts } from '../db/query';
 
+// Is there a reason why this isn't just being pulled dynamically from https://cloud.google.com/compute/docs/reference/rest/v1/machineTypes/list ?
```
Comment (hugorut) [inline src/scrapers/gcpMachineTypes.ts:10]: @aliscott can you shed any light as to why we're not calling this endpoint? Was it just not available when this was written?

@infowolfe can we just remove this comment from the PR changes 🙏

**Review by hugorut — APPROVED** [review]
Thanks @infowolfe, this LGTM, can we just cleanup the comments and then I'll merge.

File: src/scrapers/gcpMachineTypes.ts (hunk 1 above)
Comment (aliscott) [reply src/scrapers/gcpMachineTypes.ts:10]: These descriptions are used to map the results from that API response to the pricing API's results. Are those descriptions returned by this API call? We already call [this API here](https://github.com/infracost/cloud-pricing-api/blob/master/src/scrapers/gcpMachineTypes.ts#L85).

Comment (hugorut) [reply src/scrapers/gcpMachineTypes.ts:10]: ah, makes sense, just ignore me then

Comment (infowolfe) [reply src/scrapers/gcpMachineTypes.ts:10]: @hugorut would you prefer that I change the comment to reference the api call instead as a TODO? This is a really naive PR that fixes the immediate problem for myself, but I'm not a ts/js developer.

Comment (hugorut) [reply src/scrapers/gcpMachineTypes.ts]: works for me

**Pull request merged**